Started GET "/events" for ::1 at 2016-12-07 16:41:39 -0500
Processing by EventsController#index as HTML
  Rendering events/index.html.erb within layouts/application
MONGODB | Topology type 'unknown' initializing.
MONGODB | Server localhost:27017 initializing.
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
Started GET "/" for ::1 at 2016-12-07 16:41:58 -0500
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
MONGODB | Connection refused - connect(2) for 127.0.0.1:27017
  Rendered events/index.html.erb within layouts/application (29742.6ms)
Completed 401 Unauthorized in 29803ms


Started GET "/events" for ::1 at 2016-12-07 16:42:24 -0500
Processing by EventsController#index as HTML
  Rendering events/index.html.erb within layouts/application
MONGODB | Topology type 'unknown' initializing.
MONGODB | Server localhost:27017 initializing.
MONGODB | Topology type 'unknown' changed to type 'single'.
MONGODB | Server description for localhost:27017 changed from 'unknown' to 'standalone'.
MONGODB | There was a change in the members of the 'single' topology.
MONGODB | localhost:27017 | go_loco_development.find | STARTED | {"find"=>"events", "filter"=>{}}
MONGODB | localhost:27017 | go_loco_development.find | SUCCEEDED | 0.001599s
  Rendered events/index.html.erb within layouts/application (44.2ms)
Completed 200 OK in 861ms (Views: 836.4ms)


Started GET "/events/new" for ::1 at 2016-12-07 16:42:27 -0500
Processing by EventsController#new as HTML
  Rendering events/new.html.erb within layouts/application
  Rendered events/_form.html.erb (45.3ms)
  Rendered events/new.html.erb within layouts/application (49.6ms)
Completed 200 OK in 97ms (Views: 91.2ms)


Started GET "/" for ::1 at 2016-12-22 10:38:38 -0500
Processing by Rails::WelcomeController#index as HTML
  Parameters: {"internal"=>true}
  Rendering /Users/LukasBarry/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/railties-5.0.0.1/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /Users/LukasBarry/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/railties-5.0.0.1/lib/rails/templates/rails/welcome/index.html.erb (7.3ms)
Completed 200 OK in 38ms (Views: 18.6ms)


Started GET "/event" for ::1 at 2016-12-22 10:38:41 -0500
  
ActionController::RoutingError (No route matches [GET] "/event"):
  
actionpack (5.0.0.1) lib/action_dispatch/middleware/debug_exceptions.rb:53:in `call'
web-console (3.4.0) lib/web_console/middleware.rb:135:in `call_app'
web-console (3.4.0) lib/web_console/middleware.rb:28:in `block in call'
web-console (3.4.0) lib/web_console/middleware.rb:18:in `catch'
web-console (3.4.0) lib/web_console/middleware.rb:18:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
railties (5.0.0.1) lib/rails/rack/logger.rb:36:in `call_app'
railties (5.0.0.1) lib/rails/rack/logger.rb:24:in `block in call'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:70:in `block in tagged'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:26:in `tagged'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:70:in `tagged'
railties (5.0.0.1) lib/rails/rack/logger.rb:24:in `call'
sprockets-rails (3.2.0) lib/sprockets/rails/quiet_assets.rb:13:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/request_id.rb:24:in `call'
rack (2.0.1) lib/rack/method_override.rb:22:in `call'
rack (2.0.1) lib/rack/runtime.rb:22:in `call'
activesupport (5.0.0.1) lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/static.rb:136:in `call'
rack (2.0.1) lib/rack/sendfile.rb:111:in `call'
railties (5.0.0.1) lib/rails/engine.rb:522:in `call'
puma (3.6.2) lib/puma/configuration.rb:225:in `call'
puma (3.6.2) lib/puma/server.rb:578:in `handle_request'
puma (3.6.2) lib/puma/server.rb:415:in `process_client'
puma (3.6.2) lib/puma/server.rb:275:in `block in run'
puma (3.6.2) lib/puma/thread_pool.rb:116:in `block in spawn_thread'
  Rendering /Users/LukasBarry/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/actionpack-5.0.0.1/lib/action_dispatch/middleware/templates/rescues/routing_error.html.erb within rescues/layout
  Rendering /Users/LukasBarry/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/actionpack-5.0.0.1/lib/action_dispatch/middleware/templates/rescues/_trace.html.erb
  Rendered /Users/LukasBarry/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/actionpack-5.0.0.1/lib/action_dispatch/middleware/templates/rescues/_trace.html.erb (2.5ms)
  Rendered collection of /Users/LukasBarry/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/actionpack-5.0.0.1/lib/action_dispatch/middleware/templates/routes/_route.html.erb [43 times] (29.7ms)
  Rendered /Users/LukasBarry/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/actionpack-5.0.0.1/lib/action_dispatch/middleware/templates/routes/_table.html.erb (15.1ms)
  Rendering /Users/LukasBarry/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/actionpack-5.0.0.1/lib/action_dispatch/middleware/templates/rescues/_request_and_response.html.erb
  Rendered /Users/LukasBarry/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/actionpack-5.0.0.1/lib/action_dispatch/middleware/templates/rescues/_request_and_response.html.erb (1.8ms)
  Rendered /Users/LukasBarry/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/actionpack-5.0.0.1/lib/action_dispatch/middleware/templates/rescues/routing_error.html.erb within rescues/layout (185.6ms)
Started GET "/events" for ::1 at 2016-12-22 10:38:49 -0500
Processing by EventsController#index as HTML
  Rendering events/index.html.erb within layouts/application
MONGODB | Topology type 'unknown' initializing.
MONGODB | Server localhost:27017 initializing.
MONGODB | Topology type 'unknown' changed to type 'single'.
MONGODB | Server description for localhost:27017 changed from 'unknown' to 'standalone'.
MONGODB | There was a change in the members of the 'single' topology.
MONGODB | localhost:27017 | go_loco_development.find | STARTED | {"find"=>"events", "filter"=>{}}
MONGODB | localhost:27017 | go_loco_development.find | SUCCEEDED | 0.014987s
  Rendered events/index.html.erb within layouts/application (85.1ms)
Completed 200 OK in 446ms (Views: 430.8ms)


Started GET "/events/new" for ::1 at 2016-12-22 10:38:56 -0500
Processing by EventsController#new as HTML
  Rendering events/new.html.erb within layouts/application
  Rendered events/_form.html.erb (31.9ms)
  Rendered events/new.html.erb within layouts/application (35.4ms)
Completed 200 OK in 113ms (Views: 108.7ms)


Started GET "/events" for ::1 at 2016-12-22 10:39:12 -0500
Processing by EventsController#index as HTML
  Rendering events/index.html.erb within layouts/application
MONGODB | localhost:27017 | go_loco_development.find | STARTED | {"find"=>"events", "filter"=>{}}
MONGODB | localhost:27017 | go_loco_development.find | SUCCEEDED | 0.002319s
  Rendered events/index.html.erb within layouts/application (9.9ms)
Completed 200 OK in 107ms (Views: 99.2ms)


Started GET "/events" for ::1 at 2016-12-26 12:08:12 -0500
Processing by EventsController#index as HTML
Overwriting existing field url in class Event.
  Rendering events/index.html.erb within layouts/application
MONGODB | Topology type 'unknown' initializing.
MONGODB | Server localhost:27017 initializing.
MONGODB | Topology type 'unknown' changed to type 'single'.
MONGODB | Server description for localhost:27017 changed from 'unknown' to 'standalone'.
MONGODB | There was a change in the members of the 'single' topology.
MONGODB | localhost:27017 | go_loco_development.find | STARTED | {"find"=>"events", "filter"=>{}}
MONGODB | localhost:27017 | go_loco_development.find | SUCCEEDED | 0.013229s
  Rendered events/index.html.erb within layouts/application (78.7ms)
Completed 200 OK in 518ms (Views: 495.9ms)


Started GET "/events/new" for ::1 at 2016-12-26 12:08:15 -0500
Processing by EventsController#new as HTML
  Rendering events/new.html.erb within layouts/application
  Rendered events/_form.html.erb (40.1ms)
  Rendered events/new.html.erb within layouts/application (43.7ms)
Completed 200 OK in 92ms (Views: 82.6ms)


Started GET "/api/users/1" for ::1 at 2017-01-01 18:12:08 -0500
  
ActionController::RoutingError (No route matches [GET] "/api/users/1"):
  
actionpack (5.0.0.1) lib/action_dispatch/middleware/debug_exceptions.rb:53:in `call'
web-console (3.4.0) lib/web_console/middleware.rb:135:in `call_app'
web-console (3.4.0) lib/web_console/middleware.rb:28:in `block in call'
web-console (3.4.0) lib/web_console/middleware.rb:18:in `catch'
web-console (3.4.0) lib/web_console/middleware.rb:18:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
railties (5.0.0.1) lib/rails/rack/logger.rb:36:in `call_app'
railties (5.0.0.1) lib/rails/rack/logger.rb:24:in `block in call'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:70:in `block in tagged'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:26:in `tagged'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:70:in `tagged'
railties (5.0.0.1) lib/rails/rack/logger.rb:24:in `call'
sprockets-rails (3.2.0) lib/sprockets/rails/quiet_assets.rb:13:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/request_id.rb:24:in `call'
rack (2.0.1) lib/rack/runtime.rb:22:in `call'
activesupport (5.0.0.1) lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/static.rb:136:in `call'
rack (2.0.1) lib/rack/sendfile.rb:111:in `call'
railties (5.0.0.1) lib/rails/engine.rb:522:in `call'
puma (3.6.2) lib/puma/configuration.rb:225:in `call'
puma (3.6.2) lib/puma/server.rb:578:in `handle_request'
puma (3.6.2) lib/puma/server.rb:415:in `process_client'
puma (3.6.2) lib/puma/server.rb:275:in `block in run'
puma (3.6.2) lib/puma/thread_pool.rb:116:in `block in spawn_thread'
Started GET "/api/v1/users/1" for ::1 at 2017-01-01 18:12:54 -0500
  
ActionController::RoutingError (No route matches [GET] "/api/v1/users/1"):
  
actionpack (5.0.0.1) lib/action_dispatch/middleware/debug_exceptions.rb:53:in `call'
web-console (3.4.0) lib/web_console/middleware.rb:135:in `call_app'
web-console (3.4.0) lib/web_console/middleware.rb:28:in `block in call'
web-console (3.4.0) lib/web_console/middleware.rb:18:in `catch'
web-console (3.4.0) lib/web_console/middleware.rb:18:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
railties (5.0.0.1) lib/rails/rack/logger.rb:36:in `call_app'
railties (5.0.0.1) lib/rails/rack/logger.rb:24:in `block in call'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:70:in `block in tagged'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:26:in `tagged'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:70:in `tagged'
railties (5.0.0.1) lib/rails/rack/logger.rb:24:in `call'
sprockets-rails (3.2.0) lib/sprockets/rails/quiet_assets.rb:13:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/request_id.rb:24:in `call'
rack (2.0.1) lib/rack/runtime.rb:22:in `call'
activesupport (5.0.0.1) lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/static.rb:136:in `call'
rack (2.0.1) lib/rack/sendfile.rb:111:in `call'
railties (5.0.0.1) lib/rails/engine.rb:522:in `call'
puma (3.6.2) lib/puma/configuration.rb:225:in `call'
puma (3.6.2) lib/puma/server.rb:578:in `handle_request'
puma (3.6.2) lib/puma/server.rb:415:in `process_client'
puma (3.6.2) lib/puma/server.rb:275:in `block in run'
puma (3.6.2) lib/puma/thread_pool.rb:116:in `block in spawn_thread'
Started GET "/admin" for ::1 at 2017-01-01 18:17:14 -0500
  
ActionController::RoutingError (No route matches [GET] "/admin"):
  
actionpack (5.0.0.1) lib/action_dispatch/middleware/debug_exceptions.rb:53:in `call'
web-console (3.4.0) lib/web_console/middleware.rb:135:in `call_app'
web-console (3.4.0) lib/web_console/middleware.rb:28:in `block in call'
web-console (3.4.0) lib/web_console/middleware.rb:18:in `catch'
web-console (3.4.0) lib/web_console/middleware.rb:18:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
railties (5.0.0.1) lib/rails/rack/logger.rb:36:in `call_app'
railties (5.0.0.1) lib/rails/rack/logger.rb:24:in `block in call'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:70:in `block in tagged'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:26:in `tagged'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:70:in `tagged'
railties (5.0.0.1) lib/rails/rack/logger.rb:24:in `call'
sprockets-rails (3.2.0) lib/sprockets/rails/quiet_assets.rb:13:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/request_id.rb:24:in `call'
rack (2.0.1) lib/rack/runtime.rb:22:in `call'
activesupport (5.0.0.1) lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/static.rb:136:in `call'
rack (2.0.1) lib/rack/sendfile.rb:111:in `call'
railties (5.0.0.1) lib/rails/engine.rb:522:in `call'
puma (3.6.2) lib/puma/configuration.rb:225:in `call'
puma (3.6.2) lib/puma/server.rb:578:in `handle_request'
puma (3.6.2) lib/puma/server.rb:415:in `process_client'
puma (3.6.2) lib/puma/server.rb:275:in `block in run'
puma (3.6.2) lib/puma/thread_pool.rb:116:in `block in spawn_thread'
Started GET "/api/v1/users" for ::1 at 2017-01-01 18:17:21 -0500
  
ActionController::RoutingError (No route matches [GET] "/api/v1/users"):
  
actionpack (5.0.0.1) lib/action_dispatch/middleware/debug_exceptions.rb:53:in `call'
web-console (3.4.0) lib/web_console/middleware.rb:135:in `call_app'
web-console (3.4.0) lib/web_console/middleware.rb:28:in `block in call'
web-console (3.4.0) lib/web_console/middleware.rb:18:in `catch'
web-console (3.4.0) lib/web_console/middleware.rb:18:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
railties (5.0.0.1) lib/rails/rack/logger.rb:36:in `call_app'
railties (5.0.0.1) lib/rails/rack/logger.rb:24:in `block in call'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:70:in `block in tagged'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:26:in `tagged'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:70:in `tagged'
railties (5.0.0.1) lib/rails/rack/logger.rb:24:in `call'
sprockets-rails (3.2.0) lib/sprockets/rails/quiet_assets.rb:13:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/request_id.rb:24:in `call'
rack (2.0.1) lib/rack/runtime.rb:22:in `call'
activesupport (5.0.0.1) lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/static.rb:136:in `call'
rack (2.0.1) lib/rack/sendfile.rb:111:in `call'
railties (5.0.0.1) lib/rails/engine.rb:522:in `call'
puma (3.6.2) lib/puma/configuration.rb:225:in `call'
puma (3.6.2) lib/puma/server.rb:578:in `handle_request'
puma (3.6.2) lib/puma/server.rb:415:in `process_client'
puma (3.6.2) lib/puma/server.rb:275:in `block in run'
puma (3.6.2) lib/puma/thread_pool.rb:116:in `block in spawn_thread'
Started GET "/users/1" for ::1 at 2017-01-01 18:18:12 -0500
  
ActionController::RoutingError (undefined method `protect_from_forgery' for ApplicationController:Class):
  
app/controllers/application_controller.rb:2:in `<class:ApplicationController>'
app/controllers/application_controller.rb:1:in `<top (required)>'
app/controllers/api/v1/users_controller.rb:1:in `<top (required)>'
Started GET "/" for ::1 at 2017-01-02 13:03:29 -0500
Processing by Rails::WelcomeController#index as HTML
  Parameters: {"internal"=>true}
  Rendering /Users/LukasBarry/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/railties-5.0.0.1/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /Users/LukasBarry/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/railties-5.0.0.1/lib/rails/templates/rails/welcome/index.html.erb (13.1ms)
Completed 200 OK in 47ms (Views: 35.7ms)


Started GET "/events" for ::1 at 2017-01-02 13:03:33 -0500
  
ActionController::RoutingError (No route matches [GET] "/events"):
  
actionpack (5.0.0.1) lib/action_dispatch/middleware/debug_exceptions.rb:53:in `call'
web-console (3.4.0) lib/web_console/middleware.rb:135:in `call_app'
web-console (3.4.0) lib/web_console/middleware.rb:28:in `block in call'
web-console (3.4.0) lib/web_console/middleware.rb:18:in `catch'
web-console (3.4.0) lib/web_console/middleware.rb:18:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
railties (5.0.0.1) lib/rails/rack/logger.rb:36:in `call_app'
railties (5.0.0.1) lib/rails/rack/logger.rb:24:in `block in call'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:70:in `block in tagged'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:26:in `tagged'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:70:in `tagged'
railties (5.0.0.1) lib/rails/rack/logger.rb:24:in `call'
sprockets-rails (3.2.0) lib/sprockets/rails/quiet_assets.rb:13:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/request_id.rb:24:in `call'
rack (2.0.1) lib/rack/runtime.rb:22:in `call'
activesupport (5.0.0.1) lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/static.rb:136:in `call'
rack (2.0.1) lib/rack/sendfile.rb:111:in `call'
railties (5.0.0.1) lib/rails/engine.rb:522:in `call'
puma (3.6.2) lib/puma/configuration.rb:225:in `call'
puma (3.6.2) lib/puma/server.rb:578:in `handle_request'
puma (3.6.2) lib/puma/server.rb:415:in `process_client'
puma (3.6.2) lib/puma/server.rb:275:in `block in run'
puma (3.6.2) lib/puma/thread_pool.rb:116:in `block in spawn_thread'
Started GET "/events/new" for ::1 at 2017-01-02 13:03:41 -0500
  
ActionController::RoutingError (No route matches [GET] "/events/new"):
  
actionpack (5.0.0.1) lib/action_dispatch/middleware/debug_exceptions.rb:53:in `call'
web-console (3.4.0) lib/web_console/middleware.rb:135:in `call_app'
web-console (3.4.0) lib/web_console/middleware.rb:28:in `block in call'
web-console (3.4.0) lib/web_console/middleware.rb:18:in `catch'
web-console (3.4.0) lib/web_console/middleware.rb:18:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
railties (5.0.0.1) lib/rails/rack/logger.rb:36:in `call_app'
railties (5.0.0.1) lib/rails/rack/logger.rb:24:in `block in call'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:70:in `block in tagged'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:26:in `tagged'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:70:in `tagged'
railties (5.0.0.1) lib/rails/rack/logger.rb:24:in `call'
sprockets-rails (3.2.0) lib/sprockets/rails/quiet_assets.rb:13:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/request_id.rb:24:in `call'
rack (2.0.1) lib/rack/runtime.rb:22:in `call'
activesupport (5.0.0.1) lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/static.rb:136:in `call'
rack (2.0.1) lib/rack/sendfile.rb:111:in `call'
railties (5.0.0.1) lib/rails/engine.rb:522:in `call'
puma (3.6.2) lib/puma/configuration.rb:225:in `call'
puma (3.6.2) lib/puma/server.rb:578:in `handle_request'
puma (3.6.2) lib/puma/server.rb:415:in `process_client'
puma (3.6.2) lib/puma/server.rb:275:in `block in run'
puma (3.6.2) lib/puma/thread_pool.rb:116:in `block in spawn_thread'
Started GET "/events" for ::1 at 2017-01-02 13:14:48 -0500
  
ActionController::RoutingError (No route matches [GET] "/events"):
  
actionpack (5.0.0.1) lib/action_dispatch/middleware/debug_exceptions.rb:53:in `call'
web-console (3.4.0) lib/web_console/middleware.rb:135:in `call_app'
web-console (3.4.0) lib/web_console/middleware.rb:28:in `block in call'
web-console (3.4.0) lib/web_console/middleware.rb:18:in `catch'
web-console (3.4.0) lib/web_console/middleware.rb:18:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
railties (5.0.0.1) lib/rails/rack/logger.rb:36:in `call_app'
railties (5.0.0.1) lib/rails/rack/logger.rb:24:in `block in call'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:70:in `block in tagged'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:26:in `tagged'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:70:in `tagged'
railties (5.0.0.1) lib/rails/rack/logger.rb:24:in `call'
sprockets-rails (3.2.0) lib/sprockets/rails/quiet_assets.rb:13:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/request_id.rb:24:in `call'
rack (2.0.1) lib/rack/runtime.rb:22:in `call'
activesupport (5.0.0.1) lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/static.rb:136:in `call'
rack (2.0.1) lib/rack/sendfile.rb:111:in `call'
railties (5.0.0.1) lib/rails/engine.rb:522:in `call'
puma (3.6.2) lib/puma/configuration.rb:225:in `call'
puma (3.6.2) lib/puma/server.rb:578:in `handle_request'
puma (3.6.2) lib/puma/server.rb:415:in `process_client'
puma (3.6.2) lib/puma/server.rb:275:in `block in run'
puma (3.6.2) lib/puma/thread_pool.rb:116:in `block in spawn_thread'
Started GET "/events/1" for ::1 at 2017-01-02 13:14:49 -0500
  
ActionController::RoutingError (undefined method `protect_from_forgery' for ApplicationController:Class):
  
app/controllers/application_controller.rb:2:in `<class:ApplicationController>'
app/controllers/application_controller.rb:1:in `<top (required)>'
app/controllers/api/v1/events_controller.rb:1:in `<top (required)>'
Started GET "/events/1" for ::1 at 2017-01-02 13:16:31 -0500
  
ActionController::RoutingError (undefined method `protect_from_forgery' for ApplicationController:Class):
  
app/controllers/application_controller.rb:2:in `<class:ApplicationController>'
app/controllers/application_controller.rb:1:in `<top (required)>'
app/controllers/api/v1/events_controller.rb:1:in `<top (required)>'
Started GET "/events/2" for ::1 at 2017-01-02 19:21:25 -0500
  
ActionController::RoutingError (undefined method `protect_from_forgery' for ApplicationController:Class):
  
app/controllers/application_controller.rb:2:in `<class:ApplicationController>'
app/controllers/application_controller.rb:1:in `<top (required)>'
app/controllers/api/v1/events_controller.rb:1:in `<top (required)>'
Started GET "/api/events/2" for ::1 at 2017-01-02 19:22:39 -0500
  
ActionController::RoutingError (No route matches [GET] "/api/events/2"):
  
actionpack (5.0.0.1) lib/action_dispatch/middleware/debug_exceptions.rb:53:in `call'
web-console (3.4.0) lib/web_console/middleware.rb:135:in `call_app'
web-console (3.4.0) lib/web_console/middleware.rb:28:in `block in call'
web-console (3.4.0) lib/web_console/middleware.rb:18:in `catch'
web-console (3.4.0) lib/web_console/middleware.rb:18:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
railties (5.0.0.1) lib/rails/rack/logger.rb:36:in `call_app'
railties (5.0.0.1) lib/rails/rack/logger.rb:24:in `block in call'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:70:in `block in tagged'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:26:in `tagged'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:70:in `tagged'
railties (5.0.0.1) lib/rails/rack/logger.rb:24:in `call'
sprockets-rails (3.2.0) lib/sprockets/rails/quiet_assets.rb:13:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/request_id.rb:24:in `call'
rack (2.0.1) lib/rack/runtime.rb:22:in `call'
activesupport (5.0.0.1) lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/static.rb:136:in `call'
rack (2.0.1) lib/rack/sendfile.rb:111:in `call'
railties (5.0.0.1) lib/rails/engine.rb:522:in `call'
puma (3.6.2) lib/puma/configuration.rb:225:in `call'
puma (3.6.2) lib/puma/server.rb:578:in `handle_request'
puma (3.6.2) lib/puma/server.rb:415:in `process_client'
puma (3.6.2) lib/puma/server.rb:275:in `block in run'
puma (3.6.2) lib/puma/thread_pool.rb:116:in `block in spawn_thread'
Started GET "/events/2" for ::1 at 2017-01-02 19:22:54 -0500
  
ActionController::RoutingError (undefined method `protect_from_forgery' for ApplicationController:Class):
  
app/controllers/application_controller.rb:2:in `<class:ApplicationController>'
app/controllers/application_controller.rb:1:in `<top (required)>'
app/controllers/api/v1/events_controller.rb:1:in `<top (required)>'
Started GET "/events/2" for ::1 at 2017-01-02 19:23:18 -0500
  
ActionController::RoutingError (undefined method `protect_from_forgery' for ApplicationController:Class):
  
app/controllers/application_controller.rb:2:in `<class:ApplicationController>'
app/controllers/application_controller.rb:1:in `<top (required)>'
app/controllers/api/v1/events_controller.rb:1:in `<top (required)>'
Started GET "/events/2" for ::1 at 2017-01-02 19:25:55 -0500
  
ActionController::RoutingError (undefined method `protect_from_forgery' for ApplicationController:Class):
  
app/controllers/application_controller.rb:2:in `<class:ApplicationController>'
app/controllers/application_controller.rb:1:in `<top (required)>'
app/controllers/api/v1/events_controller.rb:1:in `<top (required)>'
Started GET "/events/2" for ::1 at 2017-01-02 19:25:55 -0500
  
ActionController::RoutingError (undefined method `protect_from_forgery' for ApplicationController:Class):
  
app/controllers/application_controller.rb:2:in `<class:ApplicationController>'
app/controllers/application_controller.rb:1:in `<top (required)>'
app/controllers/api/v1/events_controller.rb:1:in `<top (required)>'
Started GET "/events/2" for ::1 at 2017-01-02 19:26:10 -0500
  
ActionController::RoutingError (undefined method `protect_from_forgery' for ApplicationController:Class):
  
app/controllers/application_controller.rb:2:in `<class:ApplicationController>'
app/controllers/application_controller.rb:1:in `<top (required)>'
app/controllers/api/v1/events_controller.rb:1:in `<top (required)>'
Started GET "/api/v1/events/2" for ::1 at 2017-01-02 19:26:28 -0500
  
ActionController::RoutingError (No route matches [GET] "/api/v1/events/2"):
  
actionpack (5.0.0.1) lib/action_dispatch/middleware/debug_exceptions.rb:53:in `call'
web-console (3.4.0) lib/web_console/middleware.rb:135:in `call_app'
web-console (3.4.0) lib/web_console/middleware.rb:28:in `block in call'
web-console (3.4.0) lib/web_console/middleware.rb:18:in `catch'
web-console (3.4.0) lib/web_console/middleware.rb:18:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
railties (5.0.0.1) lib/rails/rack/logger.rb:36:in `call_app'
railties (5.0.0.1) lib/rails/rack/logger.rb:24:in `block in call'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:70:in `block in tagged'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:26:in `tagged'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:70:in `tagged'
railties (5.0.0.1) lib/rails/rack/logger.rb:24:in `call'
sprockets-rails (3.2.0) lib/sprockets/rails/quiet_assets.rb:13:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/request_id.rb:24:in `call'
rack (2.0.1) lib/rack/runtime.rb:22:in `call'
activesupport (5.0.0.1) lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/static.rb:136:in `call'
rack (2.0.1) lib/rack/sendfile.rb:111:in `call'
railties (5.0.0.1) lib/rails/engine.rb:522:in `call'
puma (3.6.2) lib/puma/configuration.rb:225:in `call'
puma (3.6.2) lib/puma/server.rb:578:in `handle_request'
puma (3.6.2) lib/puma/server.rb:415:in `process_client'
puma (3.6.2) lib/puma/server.rb:275:in `block in run'
puma (3.6.2) lib/puma/thread_pool.rb:116:in `block in spawn_thread'
Started GET "/events/2" for ::1 at 2017-01-03 08:21:06 -0500
  
SyntaxError (/Users/LukasBarry/Desktop/programming/tech_talent_south/rails/go_loco/app/controllers/api/v1/events_controller.rb:5: syntax error, unexpected '}', expecting =>
    respond_with db.Event.find( { '_id' } )
                                         ^
/Users/LukasBarry/Desktop/programming/tech_talent_south/rails/go_loco/app/controllers/api/v1/events_controller.rb:13: syntax error, unexpected end-of-input, expecting keyword_end):
  
app/controllers/api/v1/events_controller.rb:5: syntax error, unexpected '}', expecting =>
app/controllers/api/v1/events_controller.rb:13: syntax error, unexpected end-of-input, expecting keyword_end
Started GET "/events/2" for ::1 at 2017-01-03 08:21:37 -0500
  
SyntaxError (/Users/LukasBarry/Desktop/programming/tech_talent_south/rails/go_loco/app/controllers/api/v1/events_controller.rb:5: syntax error, unexpected '}', expecting =>
    respond_with db.Event.find( { '_id' } )\n
                                         ^):
  
app/controllers/api/v1/events_controller.rb:5: syntax error, unexpected '}', expecting =>
Started GET "/events/2" for ::1 at 2017-01-03 08:21:46 -0500
  
SyntaxError (/Users/LukasBarry/Desktop/programming/tech_talent_south/rails/go_loco/app/controllers/api/v1/events_controller.rb:5: syntax error, unexpected '}', expecting =>
    respond_with db.event.find( { '_id' } )
                                         ^
/Users/LukasBarry/Desktop/programming/tech_talent_south/rails/go_loco/app/controllers/api/v1/events_controller.rb:13: syntax error, unexpected end-of-input, expecting keyword_end):
  
app/controllers/api/v1/events_controller.rb:5: syntax error, unexpected '}', expecting =>
app/controllers/api/v1/events_controller.rb:13: syntax error, unexpected end-of-input, expecting keyword_end
Started GET "/events/2" for ::1 at 2017-01-03 09:36:08 -0500
  
SyntaxError (/Users/LukasBarry/Desktop/programming/tech_talent_south/rails/go_loco/app/controllers/api/v1/events_controller.rb:5: syntax error, unexpected '}', expecting =>
    respond_with db.event.find( { '_id' } )
                                         ^
/Users/LukasBarry/Desktop/programming/tech_talent_south/rails/go_loco/app/controllers/api/v1/events_controller.rb:13: syntax error, unexpected end-of-input, expecting keyword_end):
  
app/controllers/api/v1/events_controller.rb:5: syntax error, unexpected '}', expecting =>
app/controllers/api/v1/events_controller.rb:13: syntax error, unexpected end-of-input, expecting keyword_end
Started GET "/events/2" for ::1 at 2017-01-03 09:36:24 -0500
  
SyntaxError (/Users/LukasBarry/Desktop/programming/tech_talent_south/rails/go_loco/app/controllers/api/v1/events_controller.rb:5: syntax error, unexpected '}', expecting =>
    respond_with db.Event.find( { '_id' } )
                                         ^
/Users/LukasBarry/Desktop/programming/tech_talent_south/rails/go_loco/app/controllers/api/v1/events_controller.rb:13: syntax error, unexpected end-of-input, expecting keyword_end):
  
app/controllers/api/v1/events_controller.rb:5: syntax error, unexpected '}', expecting =>
app/controllers/api/v1/events_controller.rb:13: syntax error, unexpected end-of-input, expecting keyword_end
Started GET "/events/2" for ::1 at 2017-01-03 09:47:13 -0500
  
ActionController::RoutingError (undefined method `protect_from_forgery' for ApplicationController:Class):
  
app/controllers/application_controller.rb:2:in `<class:ApplicationController>'
app/controllers/application_controller.rb:1:in `<top (required)>'
app/controllers/api/v1/events_controller.rb:1:in `<top (required)>'
Started GET "/events/2" for ::1 at 2017-01-03 09:52:53 -0500
  
ActionController::RoutingError (undefined method `protect_from_forgery' for ApplicationController:Class):
  
app/controllers/application_controller.rb:2:in `<class:ApplicationController>'
app/controllers/application_controller.rb:1:in `<top (required)>'
app/controllers/api/v1/events_controller.rb:1:in `<top (required)>'
Started GET "/events/2" for ::1 at 2017-01-03 09:53:01 -0500
  
ActionController::RoutingError (undefined method `protect_from_forgery' for ApplicationController:Class):
  
app/controllers/application_controller.rb:2:in `<class:ApplicationController>'
app/controllers/application_controller.rb:1:in `<top (required)>'
app/controllers/api/v1/events_controller.rb:1:in `<top (required)>'
Started GET "/events/2" for ::1 at 2017-01-03 09:53:53 -0500
  
ActionController::RoutingError (undefined method `protect_from_forgery' for ApplicationController:Class):
  
app/controllers/application_controller.rb:2:in `<class:ApplicationController>'
app/controllers/application_controller.rb:1:in `<top (required)>'
app/controllers/api/v1/events_controller.rb:1:in `<top (required)>'
Started GET "/events/2" for ::1 at 2017-01-03 09:54:35 -0500
  
ActionController::RoutingError (undefined method `protect_from_forgery' for ApplicationController:Class):
  
app/controllers/application_controller.rb:2:in `<class:ApplicationController>'
app/controllers/application_controller.rb:1:in `<top (required)>'
app/controllers/api/v1/events_controller.rb:1:in `<top (required)>'
Started GET "/events/2" for ::1 at 2017-01-03 09:56:14 -0500
  
LoadError (Unable to autoload constant Api::V1::EventsController, expected /Users/LukasBarry/Desktop/programming/tech_talent_south/rails/go_loco/app/controllers/api/v1/events_controller.rb to define it):
  
activesupport (5.0.0.1) lib/active_support/dependencies.rb:512:in `load_missing_constant'
activesupport (5.0.0.1) lib/active_support/dependencies.rb:203:in `const_missing'
activesupport (5.0.0.1) lib/active_support/inflector/methods.rb:270:in `const_get'
activesupport (5.0.0.1) lib/active_support/inflector/methods.rb:270:in `block in constantize'
activesupport (5.0.0.1) lib/active_support/inflector/methods.rb:266:in `each'
activesupport (5.0.0.1) lib/active_support/inflector/methods.rb:266:in `inject'
activesupport (5.0.0.1) lib/active_support/inflector/methods.rb:266:in `constantize'
activesupport (5.0.0.1) lib/active_support/dependencies.rb:583:in `get'
activesupport (5.0.0.1) lib/active_support/dependencies.rb:614:in `constantize'
actionpack (5.0.0.1) lib/action_dispatch/http/request.rb:93:in `controller_class'
actionpack (5.0.0.1) lib/action_dispatch/routing/route_set.rb:44:in `controller'
actionpack (5.0.0.1) lib/action_dispatch/routing/route_set.rb:30:in `serve'
actionpack (5.0.0.1) lib/action_dispatch/routing/mapper.rb:16:in `block in <class:Constraints>'
actionpack (5.0.0.1) lib/action_dispatch/routing/mapper.rb:46:in `serve'
actionpack (5.0.0.1) lib/action_dispatch/journey/router.rb:39:in `block in serve'
actionpack (5.0.0.1) lib/action_dispatch/journey/router.rb:26:in `each'
actionpack (5.0.0.1) lib/action_dispatch/journey/router.rb:26:in `serve'
actionpack (5.0.0.1) lib/action_dispatch/routing/route_set.rb:725:in `call'
warden (1.2.6) lib/warden/manager.rb:35:in `block in call'
warden (1.2.6) lib/warden/manager.rb:34:in `catch'
warden (1.2.6) lib/warden/manager.rb:34:in `call'
rack (2.0.1) lib/rack/etag.rb:25:in `call'
rack (2.0.1) lib/rack/conditional_get.rb:25:in `call'
rack (2.0.1) lib/rack/head.rb:12:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/callbacks.rb:38:in `block in call'
activesupport (5.0.0.1) lib/active_support/callbacks.rb:97:in `__run_callbacks__'
activesupport (5.0.0.1) lib/active_support/callbacks.rb:750:in `_run_call_callbacks'
activesupport (5.0.0.1) lib/active_support/callbacks.rb:90:in `run_callbacks'
actionpack (5.0.0.1) lib/action_dispatch/middleware/callbacks.rb:36:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/remote_ip.rb:79:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/debug_exceptions.rb:49:in `call'
web-console (3.4.0) lib/web_console/middleware.rb:135:in `call_app'
web-console (3.4.0) lib/web_console/middleware.rb:28:in `block in call'
web-console (3.4.0) lib/web_console/middleware.rb:18:in `catch'
web-console (3.4.0) lib/web_console/middleware.rb:18:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
railties (5.0.0.1) lib/rails/rack/logger.rb:36:in `call_app'
railties (5.0.0.1) lib/rails/rack/logger.rb:24:in `block in call'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:70:in `block in tagged'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:26:in `tagged'
activesupport (5.0.0.1) lib/active_support/tagged_logging.rb:70:in `tagged'
railties (5.0.0.1) lib/rails/rack/logger.rb:24:in `call'
sprockets-rails (3.2.0) lib/sprockets/rails/quiet_assets.rb:13:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/request_id.rb:24:in `call'
rack (2.0.1) lib/rack/runtime.rb:22:in `call'
activesupport (5.0.0.1) lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.0.1) lib/action_dispatch/middleware/static.rb:136:in `call'
rack (2.0.1) lib/rack/sendfile.rb:111:in `call'
railties (5.0.0.1) lib/rails/engine.rb:522:in `call'
puma (3.6.2) lib/puma/configuration.rb:225:in `call'
puma (3.6.2) lib/puma/server.rb:578:in `handle_request'
puma (3.6.2) lib/puma/server.rb:415:in `process_client'
puma (3.6.2) lib/puma/server.rb:275:in `block in run'
puma (3.6.2) lib/puma/thread_pool.rb:116:in `block in spawn_thread'
Started GET "/events/2" for ::1 at 2017-01-03 09:57:24 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"2"}
Completed 500 Internal Server Error in 82ms


  
NameError (undefined local variable or method `db' for #<Api::V1::EventsController:0x007fca79d32b88>):
  
app/controllers/api/v1/events_controller.rb:5:in `show'
Started GET "/events/2" for ::1 at 2017-01-03 09:57:47 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"2"}
Overwriting existing field url in class Event.
MONGODB | Topology type 'unknown' initializing.
MONGODB | Server localhost:27017 initializing.
MONGODB | Topology type 'unknown' changed to type 'single'.
MONGODB | Server description for localhost:27017 changed from 'unknown' to 'standalone'.
MONGODB | There was a change in the members of the 'single' topology.
MONGODB | localhost:27017 | go_loco_development.find | STARTED | {"find"=>"events", "filter"=>{"_id"=>{"_id"=>"2"}}}
MONGODB | localhost:27017 | go_loco_development.find | SUCCEEDED | 0.0017050000000000001s
Completed 404 Not Found in 76ms


  
Mongoid::Errors::DocumentNotFound (
message:
  Document(s) not found for class Event with id(s) {"_id"=>"2"}.
summary:
  When calling Event.find with an id or array of ids, each parameter must match a document in the database or this error will be raised. The search was for the id(s): {"_id"=>"2"} ... (1 total) and the following ids were not found: {"_id"=>"2"}.
resolution:
  Search for an id that is in the database or set the Mongoid.raise_not_found_error configuration option to false, which will cause a nil to be returned instead of raising this error when searching for a single id, or only the matched documents when searching for multiples.):
  
app/controllers/api/v1/events_controller.rb:5:in `show'
Started GET "/events/2" for ::1 at 2017-01-03 10:15:42 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"2"}
MONGODB | localhost:27017 | go_loco_development.find | STARTED | {"find"=>"events", "filter"=>{"_id"=>{"_id"=>"2"}}}
MONGODB | localhost:27017 | go_loco_development.find | SUCCEEDED | 0.00353s
Completed 404 Not Found in 23ms


  
Mongoid::Errors::DocumentNotFound (
message:
  Document(s) not found for class Event with id(s) {"_id"=>"2"}.
summary:
  When calling Event.find with an id or array of ids, each parameter must match a document in the database or this error will be raised. The search was for the id(s): {"_id"=>"2"} ... (1 total) and the following ids were not found: {"_id"=>"2"}.
resolution:
  Search for an id that is in the database or set the Mongoid.raise_not_found_error configuration option to false, which will cause a nil to be returned instead of raising this error when searching for a single id, or only the matched documents when searching for multiples.):
  
app/controllers/api/v1/events_controller.rb:5:in `show'
Started GET "/events/2" for ::1 at 2017-01-03 10:21:30 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"2"}
Overwriting existing field url in class Event.
Completed 500 Internal Server Error in 17ms


  
Mongoid::Errors::InvalidFind (
message:
  Calling Document.find with nil is invalid.
summary:
  Document.find expects the parameters to be 1 or more ids, and will return a single document if 1 id is provided, otherwise an array of documents if multiple ids are provided.
resolution:
  Most likely this is caused by passing parameters directly through to the find, and the parameter either is not present or the key from which it is accessed is incorrect.):
  
app/controllers/api/v1/events_controller.rb:5:in `show'
Started GET "/events/2" for ::1 at 2017-01-03 10:21:42 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"2"}
Overwriting existing field url in class Event.
Completed 500 Internal Server Error in 12ms


  
Mongoid::Errors::InvalidFind (
message:
  Calling Document.find with nil is invalid.
summary:
  Document.find expects the parameters to be 1 or more ids, and will return a single document if 1 id is provided, otherwise an array of documents if multiple ids are provided.
resolution:
  Most likely this is caused by passing parameters directly through to the find, and the parameter either is not present or the key from which it is accessed is incorrect.):
  
app/controllers/api/v1/events_controller.rb:5:in `show'
Started GET "/events/2" for ::1 at 2017-01-03 10:22:12 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"2"}
Overwriting existing field url in class Event.
Completed 500 Internal Server Error in 13ms


  
Mongoid::Errors::InvalidFind (
message:
  Calling Document.find with nil is invalid.
summary:
  Document.find expects the parameters to be 1 or more ids, and will return a single document if 1 id is provided, otherwise an array of documents if multiple ids are provided.
resolution:
  Most likely this is caused by passing parameters directly through to the find, and the parameter either is not present or the key from which it is accessed is incorrect.):
  
app/controllers/api/v1/events_controller.rb:5:in `show'
Started GET "/events/2" for ::1 at 2017-01-03 10:22:25 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"2"}
Overwriting existing field url in class Event.
Completed 500 Internal Server Error in 10ms


  
Mongoid::Errors::InvalidFind (
message:
  Calling Document.find with nil is invalid.
summary:
  Document.find expects the parameters to be 1 or more ids, and will return a single document if 1 id is provided, otherwise an array of documents if multiple ids are provided.
resolution:
  Most likely this is caused by passing parameters directly through to the find, and the parameter either is not present or the key from which it is accessed is incorrect.):
  
app/controllers/api/v1/events_controller.rb:5:in `show'
Started GET "/events/2" for ::1 at 2017-01-03 10:23:07 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"2"}
Overwriting existing field url in class Event.
Completed 500 Internal Server Error in 11ms


  
Mongoid::Errors::InvalidFind (
message:
  Calling Document.find with nil is invalid.
summary:
  Document.find expects the parameters to be 1 or more ids, and will return a single document if 1 id is provided, otherwise an array of documents if multiple ids are provided.
resolution:
  Most likely this is caused by passing parameters directly through to the find, and the parameter either is not present or the key from which it is accessed is incorrect.):
  
app/controllers/api/v1/events_controller.rb:5:in `show'
Started GET "/events/2" for ::1 at 2017-01-03 10:24:59 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"2"}
Overwriting existing field url in class Event.
Completed 500 Internal Server Error in 10ms


  
Mongoid::Errors::InvalidFind (
message:
  Calling Document.find with nil is invalid.
summary:
  Document.find expects the parameters to be 1 or more ids, and will return a single document if 1 id is provided, otherwise an array of documents if multiple ids are provided.
resolution:
  Most likely this is caused by passing parameters directly through to the find, and the parameter either is not present or the key from which it is accessed is incorrect.):
  
app/controllers/api/v1/events_controller.rb:5:in `show'
Started GET "/events/2" for ::1 at 2017-01-03 10:25:06 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"2"}
Completed 500 Internal Server Error in 1ms


  
Mongoid::Errors::InvalidFind (
message:
  Calling Document.find with nil is invalid.
summary:
  Document.find expects the parameters to be 1 or more ids, and will return a single document if 1 id is provided, otherwise an array of documents if multiple ids are provided.
resolution:
  Most likely this is caused by passing parameters directly through to the find, and the parameter either is not present or the key from which it is accessed is incorrect.):
  
app/controllers/api/v1/events_controller.rb:5:in `show'
Started GET "/events/2" for ::1 at 2017-01-03 10:26:12 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"2"}
Overwriting existing field url in class Event.
Completed 500 Internal Server Error in 8ms


  
Mongoid::Errors::InvalidFind (
message:
  Calling Document.find with nil is invalid.
summary:
  Document.find expects the parameters to be 1 or more ids, and will return a single document if 1 id is provided, otherwise an array of documents if multiple ids are provided.
resolution:
  Most likely this is caused by passing parameters directly through to the find, and the parameter either is not present or the key from which it is accessed is incorrect.):
  
app/controllers/api/v1/events_controller.rb:5:in `show'
Started GET "/events/2" for ::1 at 2017-01-03 10:27:01 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"2"}
Overwriting existing field url in class Event.
[active_model_serializers] Rendered ActiveModel::Serializer::Null with Class (0.15ms)
Completed 200 OK in 56ms (Views: 48.4ms)


Started GET "/events/2" for ::1 at 2017-01-03 10:27:26 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"2"}
Overwriting existing field url in class Event.
MONGODB | localhost:27017 | go_loco_development.find | STARTED | {"find"=>"events", "filter"=>{"_id"=>{"_id"=>2}}}
MONGODB | localhost:27017 | go_loco_development.find | SUCCEEDED | 0.000753s
Completed 404 Not Found in 10ms


  
Mongoid::Errors::DocumentNotFound (
message:
  Document(s) not found for class Event with id(s) {:_id=>2}.
summary:
  When calling Event.find with an id or array of ids, each parameter must match a document in the database or this error will be raised. The search was for the id(s): {:_id=>2} ... (1 total) and the following ids were not found: {:_id=>2}.
resolution:
  Search for an id that is in the database or set the Mongoid.raise_not_found_error configuration option to false, which will cause a nil to be returned instead of raising this error when searching for a single id, or only the matched documents when searching for multiples.):
  
app/controllers/api/v1/events_controller.rb:5:in `show'
Started GET "/events/2" for ::1 at 2017-01-03 10:28:39 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"2"}
Overwriting existing field url in class Event.
MONGODB | localhost:27017 | go_loco_development.find | STARTED | {"find"=>"events", "filter"=>{"_id"=>{"_id"=>2}}}
MONGODB | localhost:27017 | go_loco_development.find | SUCCEEDED | 0.000686s
Completed 404 Not Found in 9ms


  
Mongoid::Errors::DocumentNotFound (
message:
  Document(s) not found for class Event with id(s) {:_id=>2}.
summary:
  When calling Event.find with an id or array of ids, each parameter must match a document in the database or this error will be raised. The search was for the id(s): {:_id=>2} ... (1 total) and the following ids were not found: {:_id=>2}.
resolution:
  Search for an id that is in the database or set the Mongoid.raise_not_found_error configuration option to false, which will cause a nil to be returned instead of raising this error when searching for a single id, or only the matched documents when searching for multiples.):
  
app/controllers/api/v1/events_controller.rb:5:in `show'
Started GET "/events/2" for ::1 at 2017-01-03 10:28:48 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"2"}
Overwriting existing field url in class Event.
[active_model_serializers] Rendered ActiveModel::Serializer::Null with Class (0.09ms)
Completed 200 OK in 17ms (Views: 8.3ms)


Started GET "/events/2" for ::1 at 2017-01-03 10:29:21 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"2"}
Overwriting existing field url in class Event.
MONGODB | localhost:27017 | go_loco_development.find | STARTED | {"find"=>"events", "filter"=>{"_id"=>"2"}}
MONGODB | localhost:27017 | go_loco_development.find | SUCCEEDED | 0.0005780000000000001s
Completed 404 Not Found in 9ms


  
Mongoid::Errors::DocumentNotFound (
message:
  Document(s) not found for class Event with id(s) 2.
summary:
  When calling Event.find with an id or array of ids, each parameter must match a document in the database or this error will be raised. The search was for the id(s): 2 ... (1 total) and the following ids were not found: 2.
resolution:
  Search for an id that is in the database or set the Mongoid.raise_not_found_error configuration option to false, which will cause a nil to be returned instead of raising this error when searching for a single id, or only the matched documents when searching for multiples.):
  
app/controllers/api/v1/events_controller.rb:5:in `show'
Started GET "/events/2" for ::1 at 2017-01-03 10:29:44 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"2"}
Overwriting existing field url in class Event.
MONGODB | localhost:27017 | go_loco_development.find | STARTED | {"find"=>"events", "filter"=>{"_id"=>"2"}}
MONGODB | localhost:27017 | go_loco_development.find | SUCCEEDED | 0.0005639999999999999s
Completed 404 Not Found in 10ms


  
Mongoid::Errors::DocumentNotFound (
message:
  Document(s) not found for class Event with id(s) 2.
summary:
  When calling Event.find with an id or array of ids, each parameter must match a document in the database or this error will be raised. The search was for the id(s): 2 ... (1 total) and the following ids were not found: 2.
resolution:
  Search for an id that is in the database or set the Mongoid.raise_not_found_error configuration option to false, which will cause a nil to be returned instead of raising this error when searching for a single id, or only the matched documents when searching for multiples.):
  
app/controllers/api/v1/events_controller.rb:5:in `show'
Started GET "/events/2" for ::1 at 2017-01-03 10:31:25 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"2"}
Overwriting existing field url in class Event.
Completed 500 Internal Server Error in 10ms


  
NoMethodError (undefined method `to_i' for :id:Symbol
Did you mean?  to_s):
  
app/controllers/api/v1/events_controller.rb:5:in `show'
Started GET "/events/2" for ::1 at 2017-01-03 10:31:48 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"2"}
Overwriting existing field url in class Event.
MONGODB | localhost:27017 | go_loco_development.find | STARTED | {"find"=>"events", "filter"=>{"_id"=>"2"}}
MONGODB | localhost:27017 | go_loco_development.find | SUCCEEDED | 0.000607s
Completed 404 Not Found in 10ms


  
Mongoid::Errors::DocumentNotFound (
message:
  Document(s) not found for class Event with id(s) 2.
summary:
  When calling Event.find with an id or array of ids, each parameter must match a document in the database or this error will be raised. The search was for the id(s): 2 ... (1 total) and the following ids were not found: 2.
resolution:
  Search for an id that is in the database or set the Mongoid.raise_not_found_error configuration option to false, which will cause a nil to be returned instead of raising this error when searching for a single id, or only the matched documents when searching for multiples.):
  
app/controllers/api/v1/events_controller.rb:5:in `show'
Started GET "/events/2" for ::1 at 2017-01-03 10:32:00 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"2"}
MONGODB | localhost:27017 | go_loco_development.find | STARTED | {"find"=>"events", "filter"=>{"_id"=>"2"}}
MONGODB | localhost:27017 | go_loco_development.find | SUCCEEDED | 0.000573s
Completed 404 Not Found in 2ms


  
Mongoid::Errors::DocumentNotFound (
message:
  Document(s) not found for class Event with id(s) 2.
summary:
  When calling Event.find with an id or array of ids, each parameter must match a document in the database or this error will be raised. The search was for the id(s): 2 ... (1 total) and the following ids were not found: 2.
resolution:
  Search for an id that is in the database or set the Mongoid.raise_not_found_error configuration option to false, which will cause a nil to be returned instead of raising this error when searching for a single id, or only the matched documents when searching for multiples.):
  
app/controllers/api/v1/events_controller.rb:5:in `show'
Started GET "/events/3" for ::1 at 2017-01-03 10:35:30 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"3"}
MONGODB | localhost:27017 | go_loco_development.find | STARTED | {"find"=>"events", "filter"=>{"_id"=>"3"}}
MONGODB | localhost:27017 | go_loco_development.find | SUCCEEDED | 0.000549s
[active_model_serializers] Rendered ActiveModel::Serializer::Null with Event (0.75ms)
Completed 200 OK in 14ms (Views: 11.8ms)


Started GET "/events/4" for ::1 at 2017-01-03 10:39:03 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"4"}
MONGODB | localhost:27017 | go_loco_development.find | STARTED | {"find"=>"events", "filter"=>{"_id"=>"4"}}
MONGODB | localhost:27017 | go_loco_development.find | SUCCEEDED | 0.000656s
Completed 404 Not Found in 3ms


  
Mongoid::Errors::DocumentNotFound (
message:
  Document(s) not found for class Event with id(s) 4.
summary:
  When calling Event.find with an id or array of ids, each parameter must match a document in the database or this error will be raised. The search was for the id(s): 4 ... (1 total) and the following ids were not found: 4.
resolution:
  Search for an id that is in the database or set the Mongoid.raise_not_found_error configuration option to false, which will cause a nil to be returned instead of raising this error when searching for a single id, or only the matched documents when searching for multiples.):
  
app/controllers/api/v1/events_controller.rb:5:in `show'
Started GET "/events/3" for ::1 at 2017-01-03 10:39:10 -0500
Processing by Api::V1::EventsController#show as JSON
  Parameters: {"id"=>"3"}
MONGODB | localhost:27017 | go_loco_development.find | STARTED | {"find"=>"events", "filter"=>{"_id"=>"3"}}
MONGODB | localhost:27017 | go_loco_development.find | SUCCEEDED | 0.000682s
[active_model_serializers] Rendered ActiveModel::Serializer::Null with Event (0.34ms)
Completed 200 OK in 3ms (Views: 0.8ms)


